tests/unit/test_utils.py

```diff
@@ -594,3 +595,28 @@ def test_construct_consistent_error_no_data():
     )
     with pytest.raises(ValueError, match=error_msg):
         _construct_consistent_values(aggregate_name, components, test_db_ag)
+
+
+def test_get_weights():
+    to_weight = simple_df.filter(variable="Emissions|CH4").timeseries()
+    maps = {(_mc, _sa): 3, (_mc, _sb): 2}
+    ret = _make_weighting_series(to_weight, maps)
+    assert len(ret) == 2
+    assert ret[to_weight.index[0]] == 3
```

Comment:
```suggestion
    assert ret[(_mc, _sa)] == 3
```

Would that a) work and if yes, b) be clearer?